DEHPEcosimPro/DstAdapter/DstAdapter.cs

```diff
@@ -0,0 +1,80 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="DstAdapter.cs" company="RHEA System S.A.">
+//    Copyright (c) 2020-2020 RHEA System S.A.
+// 
+//    Author: Sam Gerené, Alex Vorobiev, Alexander van Delft, Nathanael Smiechowski.
+// 
+//    This file is part of DEHPEcosimPro
+// 
+//    The DEHPEcosimPro is free software; you can redistribute it and/or
+//    modify it under the terms of the GNU Lesser General Public
+//    License as published by the Free Software Foundation; either
+//    version 3 of the License, or (at your option) any later version.
+// 
+//    The DEHPEcosimPro is distributed in the hope that it will be useful,
+//    but WITHOUT ANY WARRANTY; without even the implied warranty of
+//    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+//    Lesser General Public License for more details.
+// 
+//    You should have received a copy of the GNU Lesser General Public License
+//    along with this program; if not, write to the Free Software Foundation,
+//    Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+// </copyright>
+// --------------------------------------------------------------------------------------------------------------------
+
+namespace DEHPEcosimPro.DstAdapter
+{
+    using System.Threading.Tasks;
+
+    using DEHPEcosimPro.Enumerator;
+    using DEHPEcosimPro.Services.OpcConnector.Interfaces;
+
+    using Opc.Ua;
+
+    using ReactiveUI;
+
+    /// <summary>
+    /// The <see cref="DstAdapter"/> takes care of retrieving data from and to EcosimPro
+    /// </summary>
+    public class DstAdapter : ReactiveObject, IDstAdapter
```

Comment:
I have the impression that this class should be called `DstController` according to the design? I don't think this needs to be a ReactiveObject, since it is not a ViewModel.